pkg/gofr/gofr.go

```diff
@@ -85,6 +85,10 @@ func New() *App {
 
 	app.httpServer = newHTTPServer(app.container, port, middleware.GetConfigs(app.Config))
 
+	if app.Config.Get("APP_ENV") == "DEBUG" {
+		app.httpServer.RegisterProfillingRoutes()
+	}
+
 	// GRPC Server
```

Comment:
Not related to current PR but

```suggestion
 	// gRPC Server
```